src/applications/accredited-representative-portal/components/common/Header/UserNav.jsx

```diff
@@ -0,0 +1,109 @@
+import React from 'react';
+import { useSelector } from 'react-redux';
+import PropTypes from 'prop-types';
+import { Link } from 'react-router-dom-v5-compat';
+import UserNavLinks from './UserNavLinks';
+import NavigationDropdown from './NavigationDropdown';
+import {
+  selectUserProfile,
+  selectIsUserLoading,
+} from '../../../selectors/user';
+import { SIGN_IN_URL } from '../../../constants';
+
+const UserNav = ({ isMobile }) => {
+  const profile = useSelector(selectUserProfile);
+  const isLoading = useSelector(selectIsUserLoading);
+
+  let content;
+
+  if (isLoading) {
+    content = (
+      <div className="loading-icon-container">
+        <va-loading-indicator
+          data-testid="user-nav-loading-icon"
+          label="Loading"
+        />
+      </div>
+    );
+  } else if (!profile) {
+    content = (
+      <Link
+        data-testid="user-nav-sign-in-link"
+        className="usa-button usa-button-primary nav__sign-in"
+        to={SIGN_IN_URL}
+      >
+        Sign in
+      </Link>
+    );
+  } else if (profile && isMobile) {
+    content = (
+      <div className="vads-u-display--flex vads-u-justify-content--center user-nav">
+        <NavigationDropdown
+          icon="person"
+          srText="toggle menu"
+          className="usa-button usa-button-secondary nav__user-btn nav__user-btn--user vads-u-color--white"
+        >
+          <UserNavLinks />
+        </NavigationDropdown>
+
+        <NavigationDropdown
+          btnText="Menu"
+          icon="menu"
+          srText="toggle menu"
+          className="usa-button usa-button-secondary nav__user-btn nav__user-btn--menu"
```

Comment:
FYI still waiting on UX for final button color, figma has a light gray button but we do not use that in our design system. Defaulting to white which is the closest color match for now.